packages/SwingSet/src/kernel/kernel.js

```diff
@@ -146,8 +146,13 @@ export default function buildKernel(
   }
   harden(testLog);
 
-  function makeVatConsole(vatID) {
-    const origConsole = makeConsole(`${debugPrefix}SwingSet:${vatID}`);
+  function makeVatConsole(kind, vatID) {
+    const origConsole = makeConsole(`${debugPrefix}SwingSet:${kind}:${vatID}`);
+    if (kind === 'ls') {
+      // LiveSlots is historically not part of the slog mechanism.
+      // TODO should this be true for the future?
```

Comment:
Probably not, my intention was that the slog captures 1: what a vat is told to do, and 2: what a vat says about its activities, and that what liveslots does is a short-term debugging question that doesn't deserve to take up space in the slog. But, if I had the circular-buffer importance-based-logging from  Foolscap (https://github.com/Agoric/agoric-sdk/issues/1318#issuecomment-661520922) here, I'd probably record a bunch of liveslots messages into it, to facilitate postmortem debugging.